### DISCUSSION TITLE
Is it a good ides to set `C_Cpp.intelliSenseCachePath` for each Folder?

### DISCUSSION THREAD

**Discussion by ynty** [microsoft/vscode-cpptools#11742] [Q&A] [answered]
For example,

```json
{
    // in .vscode/settings.json
    "C_Cpp.intelliSenseCachePath": "${workspaceRoot}/.cache/IntelliSense",
}
```

**Comment by sean-mcmanus** [comment, marked as answer]
It's up to you. By default, the intelliSenseCachePath is a shared location so the intelliSenseCacheSize (default of 5 GB) will be shared by all workspace folders. If you have it set with a path that uses `${workspaceFolder}` then it won't be shared, so it could potentially use more disk space unless you lowered the C_Cpp.intelliSenseCacheSize.
